### PR TITLE
[FIX] tax_payer_type_label and document_types order

### PR DESCRIPTION
**Pull request by suv-bmya**

### CODE REVIEW

**Comment by Danisan** [comment]
@jco-odoo @ren-odoo This is the correction for label and document type order.

**Comment by ren-odoo** [comment]
Thank you @Danisan,

@jco-odoo I tested and it's working properly, it just needs your approval.

File: addons/l10n_cl/data/l10n_latam.document.type.csv [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
-dc_fichc,440,806,Ficha ChileCompra,FCHC,,FCHC,base.cl,False
-dc_dus,450,807,DUS,DUS,,DUS,base.cl,False
-dc_bl_cemb,460,808,B/L (Conocimiento de embarque),B/L,,B/L,base.cl,False
-dc_awb,470,809,AWB Airway Bill,AWB,,AWB,base.cl,False
-dc_mic_dta,480,810,MIC/DTA,MDT,,MDT,base.cl,False
-dc_cpor,490,811,Carta de Porte,CPR,,CPR,base.cl,False
-dc_res_sna,500,812,Resolución del SNA donde califica Servicios de Exportación,RSN,,RSN,base.cl,False
-dc_pasap,510,813,Pasaporte,PSP,,PSP,base.cl,False
-dc_cd_bol,520,814,Certificado de Depósito Bolsa Prod. Chile.,CRTD,,CRTD,base.cl,False
-dc_vp_pren,530,815,Vale de Prenda Bolsa Prod. Chile,VLPR,,VLPR,base.cl,False
```
Comment (jco-odoo) [inline addons/l10n_cl/data/l10n_latam.document.type.csv]: I am a bit confused by the change of the sequence here.  Why not put the file in the right order and do with the sequence as before?

File: addons/l10n_cl/views/res_partner.xml [hunk 2]
```diff
@@ -15,8 +15,11 @@
             <field name="state_id" position="attributes">
                 <attribute name="placeholder">Región</attribute>
             </field>
-            <field name="country_id" position="after">
-                <field name="l10n_cl_sii_taxpayer_type"/>
+            <field name="vat" position="before">
+                <label for="l10n_cl_sii_taxpayer_type" string="Tax Payer Type"/>
```
Comment (jco-odoo) [inline addons/l10n_cl/views/res_partner.xml]: Is this <label for=... really needed?  If it is not in the address or in a special nolabel div, it should put the label automatically.

File: addons/l10n_cl/data/l10n_latam.document.type.csv (hunk 1 above)
Comment (Danisan) [reply addons/l10n_cl/data/l10n_latam.document.type.csv]: It is important to us, by definition, to maintain the new sequence just for the enabled documents. By now it is irrelevant to hold a sequence for the rest of the document (at least by now). 
We made only a change in the file, which is now ordered by sequence just for visibility purpose, but yet, preserving the new sequence. This has been made at 8bb92fc4e2a

File: addons/l10n_cl/views/res_partner.xml (hunk 2 above)
Comment (Danisan) [reply addons/l10n_cl/views/res_partner.xml]: Yes @jco-odoo. This is without label:


Captura de pantalla 2019-11-14 20 42 46



and this is with label:

Captura de pantalla 2019-11-14 20 40 45

The xpath location is positioned before vat field.

Comment (jco-odoo) [reply addons/l10n_cl/views/res_partner.xml]: Then I prefer to do it like in Argentina.  (have the same logic): 
```

            <field name="vat" position="after">
                <field name="l10n_ar_afip_responsibility_type_id" string="AFIP Responsibility" options="{'no_open': True, 'no_create': True}" attrs="{'readonly': [('parent_id','!=',False)]}"/>
            </field>
```

Comment (jco-odoo) [reply addons/l10n_cl/views/res_partner.xml]: Also the readonly parent_id != False should be added

File: addons/l10n_cl/data/l10n_latam.document.type.csv (hunk 1 above)
Comment (jco-odoo) [reply addons/l10n_cl/data/l10n_latam.document.type.csv]: ok

**Comment by jco-odoo** [comment]
@ren-odoo

**Comment by jco-odoo** [comment]
@Danisan

**Comment by ren-odoo** [comment]
@Danisan thank you for the change, I tested and it's working correctly. 

cc. @jco-odoo

**Pull request closed without merging**